examples/run_example.sh

```diff
@@ -0,0 +1,14 @@
+pwd
+
+python server.py
+
+TEST_EXIT_CODE=`python client.py`
+wait $TEST_EXIT_CODE
+
+if [ -z ${TEST_EXIT_CODE+x} ] || [ "$TEST_EXIT_CODE" -ne 0 ] ; then
+  printf "${RED}Tests Failed${NC} - Exit Code: $TEST_EXIT_CODE\n"
+else
+  printf "${GREEN}Tests Passed${NC}\n"
+fi
+
+exit $TEST_EXIT_CODE
```

Comment:
How does this file work correctly if the server process blocks? When would the client actually run?

Reply:
Good catch, I posted a revision for this.  Should work correctly now.  not sure why it was working on my machine before....

Reply:
This change works, but it leaves the server process running in the background. Maybe consider something like this, to clean up the server:

```
python server.py &
SERVER_PID=$!

python client.py &
CLIENT_PID=$!

wait $CLIENT_PID
TEST_EXIT_CODE=$?

kill $SERVER_PID
```

Reply:
The server process is still left running after the test ends.